src/scatra/4C_scatra_timint_hdg.cpp

```diff
@@ -47,7 +47,6 @@ ScaTra::TimIntHDG::TimIntHDG(const std::shared_ptr<Core::FE::Discretization>& ac
       padapterrorbase_(params->get<double>("PADAPTERRORBASE")),
       padaptdegreemax_(params->get<int>("PADAPTDEGREEMAX")),
       elementdegree_(nullptr)
-
```

Comment:
There are some spurious formatting changes in this PR. They are an improvement, so I am OK with keeping them.